astropy/units/core.py

```diff
@@ -1089,12 +1089,33 @@ def get_err_str(unit):
 
         raise UnitConversionError(f"{unit_str} and {other_str} are not convertible")
 
-    def _get_converter(self, other, equivalencies=[]):
-        """Get a converter for values in ``self`` to ``other``.
+    def get_converter(self, other, equivalencies=[]):
+        """
+        Return the conversion function to convert values from ``self``
```

Comment:
Trying to think if we can get this to fit on a shorter line, maybe
```
Create a function that converts values from this unit to another.
```
(if so, also for `StructuredUnit`)